include/pam_interface/real/pamy1/nifpga/4dofs/robot.hpp

```diff
@@ -2,6 +2,20 @@
 
 #include <string>
 #include "generated.h"
+#include "pam_configuration/pam_configuration.hpp"
+
+#define NI_BIT_FILE_SUBDIR "pam_interface/pamy1/bitfile/pam_robot.lvbitx"
+
+// NI fpga configuration
+const std::string get_ni_bit_file_path()
+{
+  return pam_configuration::get_path().string()+std::string(NI_BIT_FILE_SUBDIR);
```

Comment:
To avoid potential issues due to missing `/`, it might be safer to first concatenate and then convert to string, i.e. something like the following (not sure if that works as is):

    (pam_configuration::get_path() / NI_BIT_FILE_SUBDIR).string()